connecting-to-testnet/building-binary.md

```diff
@@ -0,0 +1,64 @@
+---
+order: 1
+---
+
+# Building and Starting a Full Node
+
+## Requirements
+
+* [Go](https://golang.org/) (version 1.15 or higher)
+* [make](https://en.wikipedia.org/wiki/Make_(software))
+* [jq](https://github.com/stedolan/jq)
+* [git](https://git-scm.com/)
+* [curl](https://github.com/curl/curl)
+
+
+## Building the Binary
+
+Clone the code repository:
+
+```
+git clone --branch v1.0.1 https://github.com/b-harvest/gravity-dex
+```
+
+Install node's binary `gaiad`:
+
+```
+cd gravity-dex && make install
+```
+
+## Initializing a Node
+
+Initialize your node (replace `YOUR_MONIKER` with any string that identifies your node):
```

Comment:
What would you think about using an env variable for YOUR_MONIKER?

Reply:
Ok, but how would we describe the process of setting an ENV variable to users. Run `export MONIKER=YOUR_MONIKER`? They will still have to manually substitute the value, except now we have a risk that their ENV variables gets reset (if they close the terminal, for example).